routes/segments_routes.py

```
@@ -1,76 +1,171 @@
+        def get_sub_segment(sub):
+            if pd.isna(sub):
+                return 'unknown'
+            sub = str(sub).lower()
+            if sub == 'basic':
+                return 'Basic'
+            elif sub == 'standard':
+                return 'Standard'
+            elif sub == 'premium':
+                return 'Premium'
+            else:
+                return 'unknown'
```

Ruff (0.11.9)

Use a dictionary instead of consecutive `if` statements

(SIM116)